content/v0.75.0-rc1.md

```diff
@@ -0,0 +1,45 @@
++++
+title = "Introducing 0.75.0 Release Candidate 1"
+date = 2020-04-22
++++
+
+![Dark Forces (Win), Paku Paku (macOS), Settlers 2 (Linux)](/3games.png)
+
+Installation instructions:
+
+- [Linux](/downloads/linux/) (or [Steam on Linux](/downloads/linux#steam))
+- [Windows](/downloads/windows/)
+- [macOS](/downloads/macos/)
+
+This release is significant for several reasons:
+
+1. Our first stable build - from this point forward, builds named
+   0.75.x will only contain bugfixes.
+
+2. Start of the new development cycle for the next release - we have a number
+   of planned features lined up already.
+
+3. The first packages we offer for download through the new project home
+   page - users won't need to have GitHub account to access our builds 
+   any more.
```

Comment:
The new project homepage makes the downloads publicly accessible, without the need for a GitHub account.